socorro/external/es/super_search_fields.py

```diff
@@ -2248,9 +2248,10 @@ def get(self):
             "type": "string",
         },
     },
+    # FIXME(willkg): Remove java_stack_trace_full after January 2020.
     "java_stack_trace_full": {
         "data_validation_type": "str",
-        "description": "Raw JavaStackTrace value.",
+        "description": "(Deprecated) Raw JavaStackTrace value.",
```

Comment:
Both fields are going to show up in supersearch for a while. I figured I'd do that so people can switch to the new one over time and search both of them.